docs/data/date-pickers/date-field/date-field.md

```diff
@@ -28,15 +28,18 @@ The value of the component can be uncontrolled or controlled.
 Learn more about the _Controlled and uncontrolled_ pattern in the [React documentation](https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components).
 :::
 
-## Customize the date format
+## Localization
 
-{{"demo": "CustomDateFormat.js"}}
+See the [Date format and localization](/x/react-date-pickers/adapters-locale/) and [Translated components](/x/react-date-pickers/localization/) documentation pages for more details.
 
-## Localization
+### Customize the date format
 
-Use the `LocalizationProvider` to change the date-library locale used in the time field.
-See the [localization documentation page](/x/react-date-pickers/localization/) for more details.
+{{"demo": "CustomDateFormat.js"}}
+
+:::info
+You can find more information about [custom formats](/x/react-date-pickers/adapters-locale/#custom-formats) in the documentation page.
```

Comment:
From what I see most of the time we use the following structure to refer to another doc page:

```suggestion
You can find more information in the [Date format and localization page](/x/react-date-pickers/adapters-locale/#custom-formats)
```

Or 

```suggestion
You can find more information in the [Date format and localization](/x/react-date-pickers/adapters-locale/#custom-formats) documentation page
```

It would be nice to be more consistent in the whole doc of course, @samuelsycamore if you have a preference :smiley:

Reply:
Or:
```suggestion
See [custom formats](/x/react-date-pickers/adapters-locale/#custom-formats) documentation for more information.
```
Avoid specifying `page`, when we are talking about a section. 👌

Reply:
True, if often use "See" as well
We even have some "Take a look"

Reply:
There is no rush for this, let's wait for Sam's input. 👌

Reply:
Thank you @samuelsycamore. 🙏 
I've changed it to `Date format and localization—Custom formats` instead of `Date localization—Custom formats` to correctly reflect the page name.